github/github.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/google/go-github/v56/github"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/oauth2"
+	"k8s.io/utils/ptr"
```

Comment:
Nit: move this import along with other `*.k8s.io` imports.

Reply:
done!